pages/securing.rst

```diff
@@ -0,0 +1,58 @@
+.. _securing:
+
+****************
+Securing Graylog
+****************
+
+Logging user activity
+=====================
+
+Graylog follows microservices architecture model where the user interface retrieves all data via a collection of REST APIs. Thus logging relevant user activity is a simple matter of enabling built-in feature called RestAccessLogFilter. It catches all requests to REST APIs, and produces an access log augmented by user information.
```

Comment:
You are correct. The way services are designed, it really resembles more client-server model. It's just twisted to talk about client-server when the actual user facing application is entirely dependent on the server - and not just the data. But I guess the browser can be the client, hehe. Anyways, I changed that part. :)